File: official/docs/c#/addresses/StrictVerifyAddress.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

try {
    Address address = await Address.Create(
        new Dictionary<string, object>() {
            { "street1", "UNDELIEVRABLE ST" },
            { "street2", "FLOOR 5" },
            { "city", "SAN FRANCISCO" },
            { "state", "CA" },
            { "zip", "94104" },
            { "country", "US" },
            { "company", "EasyPost" },
            { "phone", "415-123-4567" },
            { "verifications", new List<string>() { "delivery" } }
        }
    );
} catch(HttpException e) {
    Console.WriteLine(e.code);
    Console.WriteLine(e.http_status);
    Console.WriteLine(e.message);
}

File: official/docs/node/addresses/strict_verify_address.js
const Easypost = require('@easypost/api');
const api = new Easypost('EASYPOST_API_KEY');

const address = new api.Address({
  verify_strict: true,
  street1: '417 MONTGOMERY ST',
  street2: 'FLOOR 5',
  city: 'SAN FRANCISCO',
  state: 'CA',
  zip: '94104',
  country: 'US',
  company: 'EasyPost',
  phone: '415-123-4567',
});

address.save().then(console.log);
